app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, \
+    NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+
+        if (
+                datetime.date.today() - visitor["vaccine"]["expiration_date"]
```

Comment:
Here you don`t need such complex construction. You can just compare dates like this: "visitor["vaccine"]["expiration_date"] < date.today()".

Reply:
Thanks! Fixed all troubles, it's really more simple and readable